matter_server/common/custom_clusters.py

```diff
@@ -55,6 +62,17 @@ def __init_subclass__(cls: ClusterAttributeDescriptor, *args, **kwargs) -> None:
         ALL_CUSTOM_ATTRIBUTES[cls.cluster_id][cls.attribute_id] = cls
 
 
+def should_poll_eve_energy(node_data: MatterNodeData) -> bool:
+    """Check if the (Eve Energy) custom attribute should be polled for state changes."""
+    if node_data.attributes.get("0/40/2") != 4874:
```

Comment:
I don't like these "magic numbers" too much, if we can have something which is based on the cluster library it would be much more readable what is happening here (and also below). :thinking:

Reply:
Well, yes and no. At least its really consolidated to one single function here for Eve.
But I'll create some constants

Reply:
adjusted